src/js/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -0,0 +1,166 @@
+              {!!it.credit_for_mil_training ? 'Yes' : 'No'}
+            </p>
+          </div>
+          <div className="historical-information list">
+            <h3>Historical Information</h3>
+            <div>
+              <p>
+                <strong>Benefit:&nbsp;</strong>
+                Post-9/11 GI Bill
+              </p>
+              <p>
+                <strong>Recipients:&nbsp;</strong>
+                {formatNumber(it.p911_recipients)}
+              </p>
+              <p>
+                <strong>Total paid (2014):&nbsp;</strong>
+                {formatCurrency(it.p911_tuition_fees)}
+              </p>
+            </div>
+            <div>
+              <p>
+                <strong>Benefit:&nbsp;</strong>
+                Yellow Ribbon
+              </p>
+              <p>
+                <strong>Recipients:&nbsp;</strong>
+                {formatNumber(it.p911_yr_recipients)}
+              </p>
+              <p>
+                <strong>Total paid (2014):&nbsp;</strong>
+                {formatCurrency(it.p911_yellow_ribbon)}
+              </p>
+            </div>
+          </div>
+          <div>
+            <h3>Institution codes</h3>
+            <p>
+              <strong>
+                <a onClick={this.props.showModal.bind(this, 'facilityCode')}>VA facility code:</a>
+                &nbsp;
+              </strong>
+              {+it.facility_code}
+            </p>
+            <p>
+              <strong>
+                <a onClick={this.props.showModal.bind(this, 'ipedsCode')}>ED IPEDS code:</a>
+                &nbsp;
+              </strong>
+              {+it.cross}
+            </p>
+            <p>
+              <strong>
+                <a onClick={this.props.showModal.bind(this, 'opeCode')}>ED OPE code:</a>
+                &nbsp;
+              </strong>
+              {+it.ope6}
+            </p>
+          </div>
+        </div>
+        <div className="medium-6 columns">
+          <div className="historical-information table">
+            <h3>Historical Information</h3>
+            <table className="usa-table-borderless">
+              <thead>
+                <tr>
+                  <th>Benefit</th>
+                  <th>Recipients</th>
+                  <th>Total paid (2014)</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th>Post-9/11 GI Bill</th>
+                  <td>{formatNumber(it.p911_recipients)}</td>
+                  <td>{formatCurrency(it.p911_tuition_fees)}</td>
+                </tr>
+                <tr>
+                  <th>Yellow Ribbon</th>
+                  <td>{formatNumber(it.p911_yr_recipients)}</td>
+                  <td>{formatCurrency(it.p911_yellow_ribbon)}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (state, props) => state;
```

Comment:
Typically if the component only needs a part of the state, you want to pare down the parts of the store you want to translate to the component's props. In this case, you're only going to be using `profile.attributes`.